client/post-editor/editor-sidebar/header.jsx

```diff
@@ -10,26 +10,49 @@ import Gridicon from 'gridicons';
  */
 import Button from 'components/button';
 import EditorPostType from 'post-editor/editor-post-type';
+import {
+	NESTED_SIDEBAR_NONE,
+	NESTED_SIDEBAR_REVISIONS,
+	NestedSidebarPropType,
+} from './util';
 
-function EditorSidebarHeader( { translate, toggleSidebar } ) {
-	return (
-		<div className="editor-sidebar__header">
+const EditorSidebarHeader = ( { nestedSidebar = NESTED_SIDEBAR_NONE, toggleSidebar, translate } ) => (
+	<div className="editor-sidebar__header">
+		{ nestedSidebar === NESTED_SIDEBAR_REVISIONS && (
+			<span>
+				<Button
+					borderless
+					className="editor-sidebar__header-title"
+					onClick={ toggleSidebar }
+					title={ translate( 'Close sidebar' ) }
+				>
+					<EditorPostType isSettings />
+				</Button>
+				<span>
+					→ { translate( 'Revisions' ) }
```

Comment:
Unless there's a good precedent, we should avoid inlined Unicode chars such as _→_ here.